dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -1,20 +1,26 @@
 <template>
-  <div class="col-3 p-3 topicLabel">
+  <div v-if="isEuTaxonomy" class="col-3 p-3 topicLabel">
     <h4 id="uploadReports" class="anchor title">Upload company reports</h4>
     <p>Please upload all relevant reports for this dataset in the PDF format.</p>
   </div>
   <!-- Select company reports -->
-  <div class="col-9 formFields">
+  <div v-if="isEuTaxonomy" class="formField">
     <h3 class="mt-0">Select company reports</h3>
-    <UploadDocumentsForm ref="uploadDocumentsForm" @documentsChanged="updateSelectedReports" :name="fileUpload" />
+    <UploadDocumentsForm ref="uploadDocumentsForm" @documentsChanged="updateSelectedReports" :name="name" />
   </div>
+  <div v-if="isEuTaxonomy" className="col-9 formFields"></div>
```

Comment:
What is this div for? Is this a copy paste error?

Reply:
I think this is here to get the correct styling for the eu taxonomy upload page

Reply:
Deprecated. Closing